inspector/an-inspector/build.gradle

```diff
@@ -39,13 +36,16 @@ ext {
     dependencies = [
         "androidx.legacy:legacy-support-v4:1.0.0",
         "com.google.guava:guava:28.0-android",
+
+        // backports java.nio.file https://github.com/henrik-lindqvist/safs/tree/master/safs-android
+        "com.llamalab.safs:safs-android:0.2.0"
     ]
 
-    buildToolsVersion = "29.0.2"
+    buildToolsVersion = "30.0.0"
 
     minSdkVersion = moduleMinSdkVersion() ?: 14
-    compileSdkVersion = 29
-    targetSdkVersion = 29
+    compileSdkVersion = 30
```

Comment:
@arblitroshani Is that the Android SDK version? I guess so. But why do we have it on the version 29 or new 30?
We are supporting Android 6.0 and up. Is it working for a Android version 6.0?

https://developer.android.com/studio/releases/platforms

Reply:
> Why do you wrote "change version to 4.8" ?

The backend introduced some special handling for the ios sdk update, and I sometimes had problems with SSO when using a version smaller than 4.8

Reply:
> And as I understand this change should never be merged to frontend master with triggerIO version 2.8.
It is only working with triggerIO version 2.9. Is this correct?

Correct. The version `3.6.0` of `sb_parse` is not compatible with Trigger 2.8
The app build would fail. 
But we're ok because the change is only made in `frontend#trigger-update-2.9`.
The release of `3.6.0` is already done. This PR only adds the new code to our plugin fork.

Reply:
> @arblitroshani Is that the Android SDK version? I guess so.

It is the Android SDK version that you use to build.

Reply:
> We are supporting Android 6.0 and up. Is it working for an Android version 6.0?

The `minSdkVersion` has not changed, so that is not affected. If previous versions worked, this will work too.
But just in case, I will try to test it in the emulator and let you know.

Reply:
@soemo Tested notifications in API 23 (Android M, 6.0)
Everything working as expected.

Reply:
Super cool thanks @arblitroshani